### PR TITLE
Implement Unattended HANA-SR Wizard Workflow

### PR DESCRIPTION
**Pull request by tomschr**
This PR contains:

* Information from the [Release Notes](https://www.suse.com/releasenotes/x86_64/SLE-SAP/12-SP4/#fate-324542); fixes FATE#325957 / Jira#sle-4237
* Changed procedure slightly (moved some sentences as separate steps).

@sknorr  I'm not sure if I should keep the Azure "thing" because it was not part of the original FATE (though the commands are the same). Azure was requested with FATE#324542 as far as I know.

### CODE REVIEW

File: xml/s4s_setup_cluster.xml [hunk 1]
```diff
@@ -656,6 +656,58 @@
...
+  <para>
+   To deploy a preconfigured &hana;-SR environments, &suse; offers an
+   unattended installation mode for the &hana;-SR wizard.
+   To use it, proceed as follows:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Create a configuration file by running the <literal>sap_ha</literal>
+     &yast; module.
```
Comment (sknorr) [inline xml/s4s_setup_cluster.xml]: I guess that needs a hint toward the manual setup section in this file.

Also not having tried this -- I wonder, do I have to fully set up HANA + HANA-SR locally before I can move my config to the cloud or how does it work? (If so, seems like a lot of effort just to move your deployment elsewhere immediately after.)

File: xml/s4s_setup_cluster.xml [hunk 2]
```diff
@@ -656,6 +656,58 @@
...
+    </para>
+   </step>
+   <step>
+    <para>
+     Save the configuration on its last screen.
+    </para>
+   </step>
+   <step>
+    <para>
+     Upload and validate the configuration automatically:
```
Comment (sknorr) [inline xml/s4s_setup_cluster.xml]: Remove "automatically".

Also, where is it uploaded to?

File: xml/s4s_setup_cluster.xml [hunk 3]
```diff
@@ -656,6 +656,58 @@
...
+    </para>
+    <screen>&prompt.root;<command>yast2 sap_ha readconfig <replaceable>CONFIGURATION_FILE_PATH</replaceable></command></screen>
+   </step>
+   <step>
+    <para>
+     Import, validate, and install the cluster unattendedly,
```
Comment (sknorr) [inline xml/s4s_setup_cluster.xml]: Do I need to switch to a different machine now?

File: xml/s4s_setup_cluster.xml [hunk 4]
```diff
@@ -656,6 +656,58 @@
...
+   &suse; is extending the &hana;-SR Wizard capabilities for the Azure Cloud.
+   Administrators can now use the &hana;-SR Wizard to configure the &hana;
+   cluster on Azure and only have to define a configuration inside the
+   wizard once and can then re-apply the saved configuration without
+   user interaction.
+  </para>
+  <para>
+   To deploy a preconfigured &hana;-SR environments, &suse; offers an
+   unattended installation mode for the &hana;-SR wizard.
+   To use it, proceed as follows:
```
Comment (sknorr) [inline xml/s4s_setup_cluster.xml]: Please mention where I need to perform these steps (what machine).

File: xml/s4s_setup_cluster.xml [hunk 5]
```diff
@@ -656,6 +656,58 @@
...
+     Import, validate, and install the cluster unattendedly,
+     based on the provided configuration file:
+    </para>
+    <screen>&prompt.root;<command>yast2 sap_ha readconfig <replaceable>CONFIGURATION_FILE_PATH</replaceable> unattended</command></screen>
+   </step>
+  </procedure>
+  <para>
+   For information on Near Zero Downtime for &hana; System Replication,
+   see also <link xlink:href="https://help.sap.com/viewer/2c1988d620e04368aa4103bf26f17727/2.0.03/en-US/ee3fd9a0c2e74733a74e4ad140fde60b.html"
+    >Use &hana; System Replication for Near Zero Downtime Upgrades</link>.
```
Comment (sknorr) [inline xml/s4s_setup_cluster.xml]: Is this relevant to the manual setup section in particular or is this more relevant in a generic For More Information section at the end of this chapter?

File: xml/s4s_setup_cluster.xml [hunk 6]
```diff
@@ -656,6 +656,58 @@
...
   </procedure>
  </sect1>
+ <sect1 xml:id="sec.s4s.hana-cluster.wizard.semiautomatic">
+  <title>Unattended Setup Using &hana;-SR Wizard</title>
+  <para>
+   Setting up a reproducible &hana; System Replication infrastructure
+   on several machines can be a lot of administrative work. The &hana;-SR
+   wizard is a manual tool that reduces the workload. However, it still
+   requires some manual steps by administrators to set it up.
+  </para>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/s4s_setup_cluster.xml]: Ok, will remove it.

File: xml/s4s_setup_cluster.xml [hunk 7]
```diff
@@ -656,6 +656,58 @@
...
+  <para>
+   &suse; is extending the &hana;-SR Wizard capabilities for the Azure Cloud.
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/s4s_setup_cluster.xml]: Ok, done.

File: xml/s4s_setup_cluster.xml [hunk 8]
```diff
@@ -656,6 +656,58 @@
...
+   Administrators can now use the &hana;-SR Wizard to configure the &hana;
+   cluster on Azure and only have to define a configuration inside the
+   wizard once and can then re-apply the saved configuration without
+   user interaction.
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/s4s_setup_cluster.xml]: * Ok, removed the `important` at the top.
* Moved the paragraph to the top.

**Comment by tomschr** [comment]
Thanks Stefan for the review! :+1: Will continue next week.

**Comment by tomschr** [comment]
As Frank suggested, merge the PR for the time being. I leave this PR open so it can be finished later.
Merged to develop with commit 3a8cfad.

**Comment by tomschr** [comment]
Added Stefan's comment in commit 6755c28

**Pull request closed without merging**